osquery/tables/utility/osquery.cpp

```diff
@@ -185,6 +185,8 @@ QueryData genOsqueryExtensions(QueryContext& context) {
 
 QueryData genOsqueryInfo(QueryContext& context) {
   QueryData results;
+
+#ifndef WIN32
```

Comment:
At the moment, it is because of the `getpid()`. This can be replaced by one of our platform abstraction functions.